pages.zh/common/docker.md

```diff
@@ -0,0 +1,36 @@
+# docker
+
+> 管理 Docker 容器和镜像。
+> 更多信息: <https://docs.docker.com/engine/reference/commandline/cli/>.
+
+- 列出目前正在运行的 docker 容器:
```

Comment:
Those aren't full-width semicolons, right? I think that's what we agreed on in #5952

Reply:
Okay.